CHANGELOG.md

```diff
@@ -3,14 +3,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## 0.3.3 - 2020-01-17
+## 0.3.3 - 02-03-2021
```

Comment:
YYYY-MM-DD is not ambiguous because nobody uses YYYY-DD-MM.

Let's change all the dates here to YYYY-MM-DD for our friends from around the world? @andi-bigswitch @Andre-Bigswitch ?

Reply:
I was going to comment the same thing. US-date-format-but-with-dashes really shouldn't exist. It's either 
`02/03/2021` or `2021-02-03`. I would vote let's use ISO because it's unambiguous (other countries, e.g., Italy use `dd/mm/yyyy`.